kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -613,6 +616,19 @@ public String toString() {
     }
   }
 
+  private static class BucketNameValidator implements ConfigDef.Validator {
```

Comment:
What do you think about moving this into the `S3BucketCheck` class, either as:
```
    public static ConfigDef.Validator bucketNameValidator() {
      return new ConfigDef.Validator() {
        ...
      };
    }
```
or moving the class into `S3BucketCheck` (still as a private static nested class) so that:
```
    public static ConfigDef.Validator bucketNameValidator() {
      return new BucketNameValidator();
    }
```